.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Jira Ticket: 
+
+# Description of Work
+Describe the work that has been done
+    
+# Acceptance Criteria
+- [x] completed acceptance criteria
+
+# Testability 
+- List of test cases
```

Comment:
You could also call this `How to test` so you don't feel like you have to describe the section in a must-remove comment.

If you can realign your headers, so there is just a dash under the sections, that would make it easier for someone to build a new pull request with the information you want. I sometimes forget to fill in an area if it started with text, so I have to remove these sections if present in templates.

Reply:
I agree with above. I would prefer a template to not have must remove comments when filling out a template. Ideally there is a header with a description of what the list should contain and then the first bullet unfilled so we know to use bullets, but don't have to erase text each pr.